### DISCUSSION TITLE
I am getting the following error on my visualizer

### DISCUSSION THREAD

**Discussion by whizbuzzer** [isl-org/Open3D#4205] [Q&A]
I have been creating a GUI for a project using the Open3D visualizer class's functionalities and have been getting the following message:
```
in __int64 __cdecl filament::UniformInterfaceBlock::getUniformOffset(const char *,unsigned __int64) const:120
reason: uniform named "srgbColor" not found
```

Does anyone know how to fix it?

**Comment by robin2002** [comment]
@whizbuzzer 
Did you solve this error? I've also encounter the same error. 
In my case, call the RemoveGeometry and AddGeometry in O3DVisualizer class for drawing 3D bounding boxes make this error.

**Comment by germanros1987** [comment]
@whizbuzzer is this issue still active?

**Comment by errissa** [comment]
@whizbuzzer Please reopen if you're still having this issue.
